lxd/daemon.go

```diff
@@ -1489,6 +1489,28 @@ func (d *Daemon) init() error {
 		}
 	}
 
+	// Set volatile.last_state.ready to false as LXD doesn't know if the instances are ready.
```

Comment:
`// Remove volatile.last_state.ready key as LXD doesn't know if the instances are ready.`

Reply:
Also please can you move this into `lxd/db/instances.go` as its own function.